test/test_scanner.py

```diff
@@ -15,18 +15,18 @@
 import tempfile
 import unittest
 import unittest.mock
-from test.test_data import __all__ as all_test_name
 from test.utils import LONG_TESTS, download_file
 
 import pytest
 
+from cve_bin_tool.checkers import __all__ as all_test_name
 from cve_bin_tool.cvedb import CVEDB
 from cve_bin_tool.version_scanner import VersionScanner
 
 BINARIES_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), "binaries")
 # load test data
 test_data = list(
-    map(lambda x: importlib.import_module(f"test.test_data.{x}"), all_test_name)
+    map(lambda x: importlib.import_module(f"test.test_data.{x}"), all_test_name[2:])
```

Comment:
`all_test_name` is sliced and 1st 2 elements are removed since the 1st 2 elements are `Checker` and `VendorProductPair`